README.md

```diff
@@ -188,12 +247,21 @@ official plugin: [eslint-plugin-testing-library][eslint-plugin-testing-library].
 [npm]: https://badge.fury.io/js/%40testing-library%2Fvue
 [license-badge]: https://img.shields.io/github/license/testing-library/vue-testing-library.svg
 [license]: https://github.com/testing-library/vue-testing-library/blob/master/LICENSE
+[discord]: https://testing-library.com/discord
```

Comment:
I'm not sure it matters but I've been using the `discord.gg` link directly on other repos 
```suggestion
[discord]: https://discord.gg/c6JN9fM
```

Reply:
it is a redirection so we should be good with both 👍